applications/ChimeraApplication/python_scripts/chimera_modelpart_import.py

```diff
@@ -0,0 +1,286 @@
+import KratosMultiphysics
+import KratosMultiphysics.ChimeraApplication as KratosChimera
+import numpy as np
+import os
+import json
+from copy import deepcopy
+
```

Comment:
I'd suggest to implement this in an OO fashion using a class to follow Kratos standards.

Reply:
This actually is a very useful feature when done in CORE. 
So I would do it in a different PR and use this utility from core in ChimeraApp.

Reply:
Agree. We can leave it for a future PR.